TrafficCapture/dockerSolution/src/main/docker/migrationConsole/workflows/templates/clusterTemplates.yaml

```diff
@@ -6,6 +6,413 @@ spec:
+                      volumeMounts:
+                        # Mount ONLY the file (ConfigMap is read-only; avoids scripts/dir creation error)
+                        - name: elasticsearch-config
+                          mountPath: /elasticsearch-2.4.0/config/elasticsearch.yml
+                          subPath: elasticsearch.yml
+                        # Writable locations
+                        - name: elasticsearch-data
+                          mountPath: /elasticsearch-2.4.0/data
+                        - name: elasticsearch-logs
+                          mountPath: /elasticsearch-2.4.0/logs
+                        # Persist the plugin installed in init container
+                        - name: plugins
+                          mountPath: /elasticsearch-2.4.0/plugins
+                  volumes:
+                    - name: elasticsearch-config
+                      configMap:
+                        name: ${CLUSTER_NAME}-config
+                    - name: elasticsearch-data
+                      emptyDir: {}
+                    - name: elasticsearch-logs
+                      emptyDir: {}
+                    - name: plugins
+                      emptyDir: {}
+            ---
+            apiVersion: v1
+            kind: Service
+            metadata:
+              name: ${CLUSTER_NAME}
+              namespace: ${NAMESPACE}
+              labels:
+                app: ${CLUSTER_NAME}
+                migration-test: "true"
+                cluster-name: ${CLUSTER_NAME}
+            spec:
+              selector:
+                app: ${CLUSTER_NAME}
+              ports:
+                - name: http
+                  port: 9200
+                  targetPort: 9200
+                - name: transport
+                  port: 9300
+                  targetPort: 9300
+            EOF
+
+            # 4) Wait for it to be available (nice UX + reliable handoff)
+            kubectl -n "${NAMESPACE}" rollout status deploy/${CLUSTER_NAME} --timeout=300s
+
+            # 5) Emit cluster config as output
+            cat > /tmp/cluster-config.json <<EOF
+            {
+              "endpoint": "http://${CLUSTER_NAME}:9200",
+              "allow_insecure": true,
+              "no_auth": null,
+              "version": "ES_2.4"
+            }
+            EOF
+
+            kubectl create configmap ${CLUSTER_NAME}-migration-config \
+              --from-file=cluster-config=/tmp/cluster-config.json \
+              -n ${NAMESPACE} \
+              --dry-run=client -o yaml | kubectl apply -f -
+
+            kubectl label configmap "${CLUSTER_NAME}-config" migration-test=true cluster-name=${CLUSTER_NAME} -n "${NAMESPACE}" --overwrite
+            kubectl label secret "${CLUSTER_NAME}-aws" migration-test=true cluster-name=${CLUSTER_NAME} -n "${NAMESPACE}" --overwrite
+            kubectl label configmap "${CLUSTER_NAME}-migration-config" migration-test=true cluster-name=${CLUSTER_NAME} -n "${NAMESPACE}" --overwrite
```

Comment:
nit: This shouldn't be necessary as we already have logic to clean this configmap up, but probably doesn't hurt to have